### ISSUE TITLE
Ubuntu-Wily mineos failing to spawn with exit status 1

### ISSUE THREAD

**Issue by educatedwarrior** [hexparrot/mineos-docker#8]
Working to get your images setup to work with FreeNAS Corral docker environment.  It should work the same ... which leads me to believe there may be an issue with your most recent dockerfile.

I setup automated docker build using a minimally modified version of your dockerfile as base image...
I just added some additional packages and planning to enable ssh. 

https://github.com/educatedwarrior/mineos-docker/blob/master/ubuntu-wily/Dockerfile 
https://hub.docker.com/r/educatedwarrior/freenas-mineos

Getting error
exited: mineos (exit status 1; not expected)

**Comment by educatedwarrior** [comment]
![4-1-2017 7-52-15 am](https://cloud.githubusercontent.com/assets/4054610/24579020/28465dce-16bb-11e7-9809-abf395c841f1.png)

**Comment by hexparrot** [comment]
I was able to build ubuntu-wily successfully just now.  Not sure what changed between then and now, but it seem to be operating as expected.

That said, having these multiple distro dockers don't really serve a purpose...nothing in the containers themselves leverage anything distro-specific, so mineos-docker is largely going to go unmaintained from here on.

I've integrated debian-jessie's dockerfile in to the hexparrot/mineos-node official repo, and it now has an autobuild on dockerhub under the name hexparrot/mineos, which should be the authoritative image from now on.
